src/sentry/api/bases/organization.py

```diff
@@ -166,7 +165,7 @@ def get_requested_project_ids(self, request):
         permission checking, use ``get_projects``, instead.
         """
         try:
-            return set(map(int, request.GET.getlist("project")))
+            return set(list(map(int, request.GET.getlist("project"))))
```

Comment:
Can we come up with a mass-transform that avoids intermediate list if we know it will end up in another collection ctor immediately?